go/client/getentries.go

```diff
@@ -28,8 +32,21 @@ func rawGetEntries(ctx context.Context, httpClient *http.Client, logURL string,
 	if end < start {
 		return nil, errors.New("start should be <= end")
 	}
+
+	baseURL, err := url.Parse(strings.TrimRight(logURL, "/") + GetEntriesPath)
+	if err != nil {
+		return nil, err
+	}
+
+	baseURL.RawQuery = url.Values{
+		"start": []string{strconv.FormatInt(start, 10)},
```

Comment:
is it generally known that FormatInt's 2nd arg is the number base?  I just had to look it up.